app/src/main/java/com/example/medicine/activity/DetailedInformationActivity.java

```diff
@@ -0,0 +1,46 @@
+package com.example.medicine.activity;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.annotation.SuppressLint;
+import android.content.Intent;
+import android.os.Bundle;
+import android.widget.TextView;
+
+import com.example.medicine.model.Card;
+import com.example.medicine.presenter.DetailedInformationContract;
+import com.example.medicine.R;
+
+public class DetailedInformationActivity extends AppCompatActivity implements com.example.medicine.view.DetailedInformationContract.view {
+
+    private com.example.medicine.view.DetailedInformationContract.presenter presenter;
```

Comment:
Надо импортировать класс, чтобы не добавлять имя пакета в этой строке. Кстати, название интерфейса `presenter` должно быть с большой буквы - видимо случайно поменялось при переименовании:
```suggestion
    private DetailedInformationContract.Presenter presenter;
```